fuse.py

```diff
@@ -754,9 +838,6 @@ def _wrapper(func, *args, **kwargs):
 
         except BaseException as e:
             self.__critical_exception = e
-            log.critical(
```

Comment:
I'd prefer to keep this in, as this keeps the logging close to the point where the exception occurred. Waiting for native FUSE to actually shut down, return the control to the calling python thread, which might then rethrow the exception (with a potentially messed-up stacktrace) bears to much potential for the exception being lost somewhere (i.e. the program crashing).

Reply:
You don't have to merge this commit but without it you get redundant exceptions. I already have an exception logger outside my call to `FUSE` so this causes things to be logged twice. I think you either have to remove re-raising `self.__critical_exception` or remove logging it here.

Why do you have a concern that the program will crash? Python programs don't crash unexpectedly. I don't see a situation where `FUSE()` doesn't return unless you're assuming the fuse library itself is buggy. Since fusepy already trusts FUSE to operate correctly in a myriad of other ways, I don't think it's consistent to assume the library is only buggy in this specific instance.

Reply:
When trying this on Windows this logging did actually come in very handy for me.
Additionally, it keeps the chain of causality, as you 'll see the exception logged before the message that FUSE is shutting down by calling `destroy`. So it is actually the reraising that might actually not be the best way to do it (I'm also still unsure what happens to the stack trace), but then so is leaving the call to FUSE without reporting what happened.
We might want to move this discussion to another PR to merge the OpenBSD support and bugfixes sooner.

Until then, if you want to keep your logs as short as possible, simply disable the "fuse" logger. ;)